src/app/navigation/example/application-launcher-example.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="padding-15">
```

Comment:
The Angular Patternfly example has a switch to hide/show the icons. Can we add that example?